### DISCUSSION TITLE
news/post/d4dabcc7-9269-46bc-a2f2-ed10f9fcf35f

### DISCUSSION THREAD

**Discussion by giscus** [SimYunSup/newsletter-ones-to-watch#2] [General]
# news/post/d4dabcc7-9269-46bc-a2f2-ed10f9fcf35f

Astro description

https://ones-to-watch.ethansup.net/news/post/d4dabcc7-9269-46bc-a2f2-ed10f9fcf35f

**Comment by ChoiWonBeen** [comment]
이벤트루프가 모든 JS실행환경의 필수적인 구현체가 아니었군요 ㄷㄷ
